app/dashboard/src/components/AriaComponents/Button/ButtonGroup.tsx

```diff
@@ -51,11 +51,11 @@ const STYLES = twv.tv({
 
 /** Props for a {@link ButtonGroup}. */
 interface ButtonGroupProps extends React.PropsWithChildren, twv.VariantProps<typeof STYLES> {
-  readonly className?: string
+  readonly className?: string | undefined
 }
 
 /** A group of buttons. */
-export function ButtonGroup(props: ButtonGroupProps) {
+function ButtonGroupInternal(props: ButtonGroupProps, ref: React.ForwardedRef<HTMLDivElement>) {
```

Comment:
Can we use the `export const ButtonGroup = React.forwardRef(function ButtonGroup(props)` format instead of this one? With this approach you'll see better display names in the devtools